composer.json

```diff
@@ -5,8 +5,12 @@
 	"keywords": ["silverstripe", "cwp"],
 	"license": "BSD-3-Clause",
 	"require": {
-		"cwp/cwp-core": "*",
-		"silverstripe/taxonomy": "*"
+		"cwp/cwp-core": "^1",
+		"silverstripe/taxonomy": "^1",
+		"silverstripe/userforms": "^4"
```

Comment:
Can you put that into the `.travis.yml` install script as well? Even if the tests require it, we don't want to prevent people  from uninstalling this module.